### PR TITLE
Open options page upon installation

### PR DESCRIPTION
**Pull request by suryatech**
Opens the options page when installed. Also fires a chrome desktop notification to draw the user's attention.

### CODE REVIEW

**Comment by suryatech** [comment]
#201: If you have wrong information in the extension, it will not let you save the modified settings. If you have not configured it for the first time, it takes you to the options page. This leaves no case in which the settings can still remain invalid.

File: bookie/static/js/bookie/view.js [hunk 1]
```diff
@@ -2038,6 +2038,13 @@ YUI.add('bookie-view', function (Y) {
                             opts.set('cache_content', 'false');
                         }
 
+                        // set the flag so that it doesn't bug you everytime
```
Comment (mitechie) [inline bookie/static/js/bookie/view.js]: Please make the comment a complete sentence.

File: bookie/static/js/bookie/view.js [hunk 2]
```diff
@@ -2038,6 +2038,13 @@ YUI.add('bookie-view', function (Y) {
...
+                        // for the configuration
+
```
Comment (mitechie) [inline bookie/static/js/bookie/view.js]: and no reason for this newline. Your comment pertains to the lines of code right after it.

File: extensions/chrome_ext/chrome.js [hunk 3]
```diff
@@ -500,6 +500,23 @@ YUI().add('bookie-chrome', function (Y) {
         init_background: function () {
             var that = this;
 
+            // Keep checking until the user configures a working
+            // combination for the first time.
+
```
Comment (mitechie) [inline extensions/chrome_ext/chrome.js]: Same thing on the comment spacing. Thanks for the nice sentence here with the period.

**Comment by mitechie** [comment]
I tried this out but it didn't work for me. I've done a build and uploaded it and then installed it but no luck. It's the 0.4.13 version.

**Comment by suryatech** [comment]
Is it on the chrome web store? Let me try it on my end. Can you kindly link to it ?

**Comment by mitechie** [comment]
Yep, it's at https://chrome.google.com/webstore/detail/bookie/knnbmilfpmbmlglpeemajjkelcbaaega

**Comment by mitechie** [comment]
As we chatted about, I can verify that the crx you sent me does work. I was going to pull down the latest code and try to build it again, but this diff is confused and has a lot of extra changes to it that make it hard for me to pull down to check. Can you look at correcting the branch or seeing if you can create a new clean pull request? Thanks so much.

**Comment by mitechie** [comment]
superceded

**Pull request closed without merging**